src/routes/__tests__/layout.test.js

```
@@ -0,0 +1,58 @@
+import { render, screen } from '@testing-library/svelte';
```

ESLint

[error] 'screen' is defined but never used.

(no-unused-vars)